_posts/announcements/2020-03-05-IGVC-Round1Qualify.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: "Round 1 Passed for Indy Autonomous Challenge"
+date: 2020-03-05
+author: Prem Bharatia
+categories: announcements
+icon: general
+---
+
+<p>
```

Comment:
This is a good, this is how content is meant to be managed with Jekyll. The projects pages use a different layout but otherwise is the same process.